src/aspire/downloader/data_fetcher.py

```diff
@@ -0,0 +1,265 @@
+import shutil
+
+import numpy as np
+import pooch
+
+from aspire import config
+from aspire.downloader import file_to_method_map, registry, registry_urls
+from aspire.volume import Volume
+
+# Initialize pooch data fetcher instance.
+data_fetcher = pooch.create(
+    # Use the default cache folder for the operating system.
+    # Pooch uses appdirs (https://github.com/ActiveState/appdirs) to
+    # select an appropriate directory for the cache on each platform.
+    path=config["common"]["cache_dir"].as_filename(),
+    # The remote data is on Zenodo base_url is a required param,
+    # even though we override using individual urls in the registry.
+    base_url="https://zenodo.org/communities/computationalcryoem/",
```

Comment:
No, we can add DOI's and other formats to the registry, which will then override this base_url.

Reply:
I see, so using a full url disregards this base_url. Okay.

Does using just the DOI in the registry instead of a full zenodo url work with this base url? Like when we go to add the new weights, I would just add DOI?...

Reply:
Yes, you can use the DOI in the registry in place of the url (using the format `doi:{DOI}/{file name}`).

The registry entry would look something like:
```
registry = {"example_FakeKV_dataset.npz": "sha_number"}
registry_urls = {"example_FakeKV_dataset.npz":  "doi:10.5281/zenodo.8186548/example_FakeKV_dataset.npz"}
```

Reply:
Sweet! Thanks!